examples/example.md

```diff
@@ -194,4 +194,4 @@ This is inline and display TeX math created using double backslashes:
 
 \\(E=mc^2\\)
 
-\\[E=mc^2\\]
\ No newline at end of file
+\\[E=mc^2\\]
```

Comment:
[We normalize the input][1], so that it always ends with a unix-style trailing `\n`. If the reader gives different results with/without trailing `\n`, that should be considered a bug.

 [1]: https://github.com/Witiko/markdown/blob/36290d5c75687462734a3d7fac29b4bdd90659a5/markdown.dtx#L24127-L24137

Reply:
Well, the output is the same, just the MarkdownLink check failed because of it.

```
examples/example.md:197:12 MD047/single-trailing-newline Files should end with a single newline character
```

Reply:
Silly MarkdownLint.